app/models/manageiq/providers/nuage/inventory/collector/target_collection.rb

```diff
@@ -117,52 +117,12 @@ def cloud_subnets_for_router(router_ref)
     safe_list { vsd_client.get_subnets_for_domain(router_ref) }
   end
 
-  def routers_for_tenant(tenant_ems_ref)
-    safe_list { vsd_client.get_domains_for_enterprise(tenant_ems_ref) }
-  end
-
-  def cloud_subnets_for_tenant(tenant_ems_ref)
-    routers_for_tenant(tenant_ems_ref).each_with_object([]) do |d, arr|
-      arr.push(safe_list { vsd_client.get_subnets_for_domain(d['ID']) })
-    end.flatten(1)
-  end
-
-  def security_groups_for_tenant(tenant_ems_ref)
-    routers_for_tenant(tenant_ems_ref).each_with_object([]) do |d, arr|
-      arr.push(safe_list { vsd_client.get_policy_groups_for_domain(d['ID']) })
-    end.flatten(1)
-  end
-
-  def tenant_ems_ref_for_cloud_subnet(cloud_subnet_ems_ref)
-    cloud_subnet = cloud_subnet(cloud_subnet_ems_ref)
-    return nil if cloud_subnet.nil?
-    router_ems_ref = zone(cloud_subnet['parentID'])['parentID']
-    network_router(router_ems_ref)['parentID']
-  end
-
-  def tenant_ems_ref_for_security_group(security_group_ems_ref)
-    security_group = security_group(security_group_ems_ref)
-    return nil if security_group.nil?
-    network_router(security_group['parentID'])['parentID']
-  end
-
   def references(collection)
     target.manager_refs_by_association.try(:[], collection).try(:[], :ems_ref).try(:to_a) || []
   end
 
   def parse_targets!
-    target.targets.each do |t|
```

Comment:
This chunk was never used anyway AFAIK since there is currently no option on UI to manually trigger targeted refresh e.g. for CloudSubnet.